app/modules/wallet/users/users.route.ts

```diff
@@ -0,0 +1,75 @@
+import * as restify from 'restify';
```

Comment:
Same comments apply here as on [this file](https://github.com/thinblock/tb-core-api/pull/11/files#diff-23144ea3fc47ac613b50c5b31c85711d)